voice.go

```diff
@@ -659,8 +658,6 @@ func (v *VoiceConnection) opusSender(udpConn *net.UDPConn, close <-chan struct{}
 		return
 	}
 
-	runtime.LockOSThread()
```

Comment:
It looks like this change was directly merged into the master branch, but never pulled into the develop branch, and it was accidentally swept up by my pull request. You should properly merge this into the develop branch, and then I can re-do my pull request and it won't show up anymore.